Dictionaries/eng_OCRFixReplaceList.xml

```diff
@@ -501,7 +496,7 @@
     <Word from="HerrSchmidt" to="Herr Schmidt" />
     <Word from="herwith" to="her with" />
     <Word from="HeY·" to="Hey." />
-    <Word from="HeyJennifer" to="Hey Jennifer" />
+    <Word from="HeyJennifer" to="Hey, Jennifer" />

```

Comment:
It's a direct address, the so-called vocative case, and it always needs a comma in English.